doc/whats_new/v1.1.rst

```diff
@@ -555,6 +555,36 @@ Changelog
   left corner of the HTML representation to show how the elements are
   clickable. :pr:`21298` by `Thomas Fan`_.
 
+Miscellaneous
+.............
+
+- |Efficiency| Low-level routines for reductions on pairwise distances
+  for dense float64 datasets have been refactored. The following functions
+  and estimators now benefit from improved performances, in particular on
+  multi-cores machines:
+  - :func:`sklearn.metrics.pairwise_distances_argmin`
+  - :func:`sklearn.metrics.pairwise_distances_argmin_min`
+  - :class:`sklearn.cluster.AffinityPropagation`
+  - :class:`sklearn.cluster.Birch`
+  - :class:`sklearn.cluster.MeanShift`
+  - :class:`sklearn.cluster.OPTICS`
+  - :class:`sklearn.cluster.SpectralClustering`
+  - :func:`sklearn.feature_selection.mutual_info_regression`
+  - :class:`sklearn.neighbors.KNeighborsClassifier`
+  - :class:`sklearn.neighbors.KNeighborsRegressor`
+  - :class:`sklearn.neighbors.LocalOutlierFactor`
+  - :class:`sklearn.neighbors.NearestNeighbors`
+  - :class:`sklearn.manifold.Isomap`
+  - :class:`sklearn.manifold.LocallyLinearEmbedding`
+  - :class:`sklearn.manifold.TSNE`
+  - :func:`sklearn.manifold.trustworthiness`
+  - :class:`sklearn.semi_supervised.LabelPropagation`
+  - :class:`sklearn.semi_supervised.LabelSpreading`
+
+  For instance :class:`sklearn.neighbors.NearestNeighbors.kneighbors`
+  can be up to ×20 faster than in the previous versions'.
+
+  :pr:`21462` by :user:`Julien Jerphanion <jjerphan>`.
```

Comment:
```suggestion
  :pr:`22288` by :user:`Julien Jerphanion <jjerphan>`.
```
Maybe, we should add the previous/prior PRs, too.

Reply:
Alternatively we can just add https://github.com/scikit-learn/scikit-learn/pull/22134.

Reply:
As you wish. It is to honor your work.

Reply:
Maybe we can link the list of all the sub PRs because they are more interesting and easier to read for historical purpose.

Reply:
Done in [5f5a83f](https://github.com/scikit-learn/scikit-learn/pull/22288/commits/5f5a83f9f3e0b89096e3eb868a49c07999f0b85b).